README.md

```diff
@@ -57,6 +57,8 @@ These instructions use the following installation paths. Be sure to substitute y
 
 <span style="background-color:#4F3C3C">**NOTE:** You can clone the project to any local directory. If you clone the project inside an existing Git repository directory (for example, the directory that contains your local O3DE engine repository) you should add the o3de-multiplayersample project directory to the Git exclude file for the existing Git repository.</span>
 
+It is expected that o3de, o3de-multiplayersample and o3de-multiplayersample-assets are all cloned from the same branch. If using development o3de-multiplayersample, then match with development branches of o3de, and o3de-multiplayersample. If using a release version of o3de, then checkout the corresponding tags for o3de-multiplayer and o3de-multiplayersample-assets ie `git checkout tags/<tag> -b local branch name>`.
```

Comment:
this is very useful info, but I feel like it's too easy to miss up here. Most folks will just jump to the steps (speaking from personal experience, anyway)

what do you think about leaving this but also adding a 5th step to both clone options like: 
"Ensure your o3de, o3de-multiplayersample, and o3de-multiplayersample-assets repos are all checked out on the same branch (such as: `development`). See above note for details."

Reply:
+1 
I'd take it even a little bit further, and just have a step after cloning where you do the git checkout and then reupdate the submodule, and put the description of the choice of branches (development vs stabilization) there.